tests/unit/s2n_alerts_test.c

```diff
@@ -274,110 +274,67 @@ int main(int argc, char **argv)
         }
     }
 
-    /* Test s2n_queue_writer_close_alert_warning */
+    /* Test s2n_alerts_write_error_or_close_notify */
     {
-        /* Safety check */
-        EXPECT_FAILURE_WITH_ERRNO(s2n_queue_writer_close_alert_warning(NULL), S2N_ERR_NULL);
-
-        /* Does not send alert if alerts not supported */
```

Comment:
Should there be a test that calls `s2n_alerts_write_error_or_close_notify` with quic enabled to test this? Or is this being tested somewhere else?